etl/steps/data/garden/animal_welfare/2023-08-14/number_of_farmed_fish.meta.yml

```diff
@@ -1,42 +1,44 @@
 definitions:
-  common: |-
-    This study estimates the annual number of farmed fish slaughtered for food using data on aquaculture production tonnages (from the Food and Agriculture Organization of the United Nations) and mean weights estimated from available fish size data (from a range of internet sources).
+  attribution: &attribution Fishcount (2018)
+  description_key: &description_key
+    - This indicator uses data on aquaculture production tonnages and mean weights estimated from available fish size data.
+    - |
+      Note that this study does not include:
+      - Non-fish species, e.g. lobsters.
+      - Fish deaths arising in fish farms prior to harvest, in the capture of wild fish for feed or re-stocking, or from environmental impacts on surrounding wildlife.
+      - Fish farmed for use as bait, whose small production tonnages may comprise large numbers of fish due to their very small size of just a few grams.
+      - Species without estimated mean weight (which could be an additional 17% of total fish production).
```

Comment:
Just wondering here: is the second element of the list a string with bullet points? If so, I guess you were trying to nest some bullet points? One thing to write down for the future: "nested lists"

Reply:
Yes, I thought the key descriptions in data pages would be able to handle this, but it doesn't. However, since there's no expectation to have a data page for this indicator, so, for now, I'll leave it like this.